tools/stimulus/functions-R.yaml

```diff
@@ -367,6 +367,10 @@ igraph_distances_johnson:
 igraph_distances_floyd_warshall:
 
 igraph_voronoi:
+    PARAMS: |-
+        GRAPH graph, OUT VECTOR_INT membership, OUT VECTOR distances,
+        VERTEX_INDICES generators, *, EDGEWEIGHTS weights=NULL, NEIMODE mode=OUT,
+        VORONOI_TIEBREAKER tiebreaker=RANDOM
```

Comment:
Seems to me this could also be a change in the C core? How can we make it easy to perhaps implement the change in `src/vendor/igraph` and then backport it to the C core repo?

Reply:
I'm not sure this will work very well in the general case, so I'm not sure adding the `*` in the C repo is a good idea. Different interfaces might want that `*` to go in different places, and then there's the convention that in C `weights` now goes directly after `graph`, but `weights` should always be keyword-only in R, so we need to handle that somehow. Perhaps with reordering like we did with `k_shortest_paths`. 

However, doing this in the C core does not affect me or the Mathematica interface, at least not at this moment.  I propose you discuss this with @ntamas, and we go ahead with whatever the two of you decide.

Merging this PR is not urgent for me.

Reply:
One option is to leave the `*` out in the C codebase and use `FIRST_KW_PARAM` in `functions-R.yaml` to tell Stimulus which parameter (by name) is the first one that is keyword-only.

If we want to do parameter reordering _and_ declaring them as keyword-only for a function, there's also an option for that: you could declare the new order in `PARAM_ORDER` _and_ insert a `*` in the parameter list on `PARAM_ORDER` at the appropriate point.

Reply:
Used `FIRST_KW_PARAM`